.github/workflows/main.yml

```diff
@@ -1,101 +1,77 @@
 name: CI
 on: [push, pull_request]
 
-permissions:
-  contents: read #  to fetch code (actions/checkout)
```

Comment:
Default to have

Reply:
I think this was about limiting the write permissions.

Reply:
OK. Let me restore this.